modules/ROOT/pages/configuring-sso.adoc

```diff
@@ -0,0 +1,435 @@
+.. Go to Microsoft Entra Id (Active Directory service), click *+ Add*, and select *Enterprise application*.
+.. In *Browse Microsoft Entra Gallary*, select the *Salesforce* application.
+.. Enter a name like *AEH Portal* and click *Create*.
+. Configure the single sign-on settings for the application:
+.. From *Manage*, select *Single sign-on* and select *SAML*. 
+.. From *Basic SAML Configuration*, click *Edit*.
+.. Complete these required fields:
++
+[%header,cols="1a,2a"]
+|===
+|Field|Value
+|Identifier (Entity ID)|https://<YOUR_DOMAIN>.my.site.com/aeh
+|Reply URL (Assertion Consumer Service URL)| https://<YOUR_DOMAIN>.my.site.com/aeh/login
+|Sign on URL |https://<YOUR_DOMAIN>.my.site.com/aeh/login
+|Logout Url (Optional)|https://<YOUR_DOMAIN>.my.site.com/aeh/services/auth/sp/saml2/logout
+|===
+.. Save your changes.
+. Configure the required first name, last name, email, and username claims that the application sends:
+.. From *Attributes & Claims*, click *Edit*.
+.. Click *Add a group claim*, select *All Groups* or any other option except *None*, and click *Save*. 
++
+At least one group must be associated to perform a group mapping in a later step.
+.. Add a new claim for each required claim by clicking *+ Add new claim* and completing these fields for each claim:
++
+* First Name
++
+[%header,cols="1a,2a"]
+|===
+|Field|Value
+|Name|firstName
+|Source attribute|user.givenname
+|===
++ 
+* Last Name
++
+[%header,cols="1a,2a"]
+|===
+|Field|Value
+|Name|lastName
+|Source attribute|user.surname
+|===
++
+* Email
++
+[%header,cols="1a,2a"]
+|===
+|Field|Value
+|Name|email
+|Source attribute|user.email
+|===
++
+* Username
++
+[%header,cols="1a,2a"]
+|===
+|Field|Value
+|Name|email
+|Source attribute|user.principalname
+|===
+.. Save your changes.
+
+*Map the Application to the Identity Provider for Anypoint Platform*
+//Note to editor: Will revisit whether to make lines 319, 321, and 323 partials if the content is still relevant once we release a single identity in SF. 
+NOTE: Skip this step if you are configuring a new identity provider for Anypoint Platform. Go to xref:configuring-sso.adoc#step-2-add-identity-providers[Step 2: Add Salesforce Identity Providers].
+
+If this single sign-on is already configured in Anypoint Platform, you must map the application to the identity provider for Anypoint Platform by adding and configuring a new custom attribute called anypoint_idp_id.   
+
+If usernames for the portal and Anypoint Platform are different, add another custom attribute named anypoint_username. This attribute updates existing identity provider users to match the identities in the portal and Anypoint Platform.
+
+. To map anypoint_idp_id to with the Anypoint Platform identity provider ID for Microsoft Azure SAML:
+
+.. Click *+ Add new claim* and complete these fields:
++
+[%header,cols="1a,2a"]
+|===
+|Field|Value
+|Name|anypoint_idp_id
+|Source attribute|<"identity_provider_id"> The ID must be in quotes.
```

Comment:
Should the quotes be outside the < > in this case, since the placeholder doesn't include the quotes (they're required)?